### PR TITLE
Volume slider for remote participant audio elements

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: modules/UI/videolayout/RemoteVideo.js [hunk 1]
```diff
@@ -132,7 +133,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let popupmenuElement = document.createElement('ul');
     popupmenuElement.className = 'popupmenu';
     popupmenuElement.id = `remote_popupmenu_${this.id}`;
-    let menuItems = [];
+    let menuLinkItems = [];
```
Comment (saghul) [inline modules/UI/videolayout/RemoteVideo.js]: Why the rename? IMHO the previous name was ok, the rename adds unnecessary noise to this PR while not adding much.

**Comment by yanas** [comment]
Lenny can you just add jsdocs for all the functions? Other than that we're ready to merge it. I'll add my styling as a separate PR.

**Comment by virtuacoplenny** [comment]
I undid the renaming and I added jsdocs. I can rebase my commits and/or squash them into one if you prefer and the changes look decent.

**Comment by virtuacoplenny** [comment]
I've squashed, rebased, and force pushed to update my branch.

**Comment by yanas** [comment]
Jenkins ok to test.

**Pull request merged**